src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -24,6 +24,7 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
 
   const posterClassnames = classNames("PosterPreview", {
     "PosterPreview--live": posterVenue.isLive,
+    "PosterPreview--related": !posterVenue.template.includes("posterpage"),
```

Comment:
Why `.includes` here? I believe `venue.template` is just a string, not an array?

```suggestion
    "PosterPreview--related": posterVenue.template !== "posterpage",
```